client/landing/gutenboarding/onboarding-block/edit.tsx

```diff
@@ -31,7 +31,7 @@ const OnboardingEdit: FunctionComponent< BlockEditProps< Attributes > > = () =>
 			<VerticalBackground />
 			{ isCreatingSite && <Redirect push to={ makePath( Step.CreateSite ) } /> }
 			<Switch>
-				<Route path={ makePath( Step.IntentGathering ) }>
+				<Route exact path={ makePath( Step.IntentGathering ) }>
```

Comment:
This route is now `/` (if no language is set), so we have to make it `exact`, since it would otherwise catch all other routes (since they're all sub-routes of `/`).